src/components/__tests__/FontAwesomeIcon.test.js

```diff
@@ -2,28 +2,63 @@
  * @jest-environment jsdom
  */
 
-import { library } from '@fortawesome/fontawesome-svg-core'
-import { faCoffee, faCircle } from '../__fixtures__/icons'
-import { compileAndMount, mountFromProps } from '../__fixtures__/helpers'
-import FontAwesomeIcon from '../FontAwesomeIcon'
+ import { library } from '@fortawesome/fontawesome-svg-core'
```

Comment:
Is there a single leading space on a bunch of these lines? It looks like it may have happened down below too.